pkg/proc/gdbserial/undo.go

```diff
@@ -324,6 +326,21 @@ func (uc *undoSession) travelToTime(p *gdbProcess, pos string) error {
 	return err
 }
 
+func (uc *undoSession) activateVolatile(p *gdbProcess) (func(), error) {
```

Comment:
Could you add a comment documenting the new method?

Reply:
:+1: